src/collections/_documentation/platforms/node/typescript.md

```diff
@@ -91,3 +91,15 @@ Sentry.init({
 ```
 
 This config should be enough to make everything work and use TypeScript with Node. And, still be able to digest all original sources by Sentry.
+
+### Dealing with TSLib
+
+During compilation, if needed, TypeScript will inject some of it's runtime dependencies into the output files it produces. It can include things like polyfills for function generators or some APIs not available in all the environments.
```

Comment:
```suggestion
During compilation, if needed, TypeScript will inject some of its runtime dependencies into the output files it produces. It can include things like polyfills for function generators or some APIs not available in all the environments.
```